pyphare/pyphare/pharesee/hierarchy/patch.py

```
@@ -67,3 +68,48 @@ def __call__(self, qty, **kwargs):
+            print("zobi : ", type(x))
+            if isinstance(x, Patch):
+                print("zob", x.patch_datas)
+                for k, v in x.patch_datas.items():
+                    print("toto", k, v, type(v))
+                    seqs.append(v)
+                    if isinstance(v, FieldData):
+                        print("is a fieldData")
+                #seqs.append(x.patch_datas)
+            else:
+                #  TODO seqs.append(x) ?
+                raise TypeError("this arg should be a Patch")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)